src/lazy/any_encoding.rs

```diff
@@ -1601,7 +1601,7 @@ impl<'data> From<LazyRawFieldExpr<'data, BinaryEncoding_1_1>>
         use LazyRawFieldExpr::*;
         match binary_field {
             NameValue(name, value) => NameValue(name.into(), value.into()),
-            NameEExp(_, _) => todo!("(name, e-exp) field in binary Ion 1.1"),
+            NameEExp(name, eexp) => NameEExp(name.into(), eexp.into()),
```

Comment:
🗺️ Converting a `LazyRawFieldExpr<_, v1_1::Binary>` to a `LazyRawFieldExpr<_, AnyEncoding>` for use in the format-agnostic `Reader`.